test/tools/ossfuzz/abiV2Proto.proto

```diff
@@ -78,6 +54,16 @@ message DynamicByteArrayType {
     required DType type = 1;
 }
 
+message ArrayType {
+    required Type t = 1;
```

Comment:
Fixes as per comment here https://github.com/ethereum/solidity/pull/6880#discussion_r300458739

For example, a two dimensional array is now encoded as ArrayType(ArrayType(int))) in place of the previous encoding (which was ArrayType(dimension:2, BaseType: int))